src/main/java/seedu/address/model/Anakin.java

```diff
@@ -141,8 +165,10 @@ public void getOutOfDeck() {
             throw new IllegalOperationWhileReviewingDeckException();
         }
         isInsideDeck = false;
-        cards = new UniqueCardList();
+        currentDeck = NULLDECK;
+        clearCards();
         updateDisplayedCards();
+        jedfalse();
```

Comment:
/nitpick can we get an alternative name for jedfalse? Took me some time to figure out what it referred to

Reply:
oh yep I was intending to refactor that